src/sidebar/services/annotations-exporter.tsx

```diff
@@ -189,18 +189,30 @@ export class AnnotationsExporter {
               </a>
             </p>
 
-            <dl>
-              <dt>Group</dt>
-              <dd>{groupName}</dd>
-              <dt>Total users</dt>
-              <dd>{uniqueUsers.length}</dd>
-              <dt>Users</dt>
-              <dd>{uniqueUsers.join(', ')}</dd>
-              <dt>Total annotations</dt>
-              <dd>{annotations.length}</dd>
-              <dt>Total replies</dt>
-              <dd>{replies.length}</dd>
-            </dl>
+            <table>
+              <tbody>
+                <tr>
+                  <td>Group:</td>
+                  <td>{groupName}</td>
+                </tr>
+                <tr>
+                  <td>Total users:</td>
+                  <td>{uniqueUsers.length}</td>
+                </tr>
+                <tr>
+                  <td>Users:</td>
+                  <td>{uniqueUsers.join(', ')}</td>
+                </tr>
+                <tr>
+                  <td>Total annotations:</td>
+                  <td>{annotations.length}</td>
+                </tr>
+                <tr>
+                  <td>Total replies:</td>
+                  <td>{replies.length}</td>
+                </tr>
+              </tbody>
+            </table>
```

Comment:
This worked well when pasted into Google Docs / Word. I do wonder whether we might want to use tables for individual annotations as well, per https://hypothes-is.slack.com/archives/C1M8NH76X/p1705057302674569. I suggest that's an exercise for a follow-up PR.

Reply:
I like it